src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_assistive.scss

```diff
@@ -0,0 +1,29 @@
+/*doc
+---
+title: Screen reader only modifier
```

Comment:
I agree, I'm willing to follow the example of twitter bootstrap here though, until I find a better way to achieve this. Do you have an Idea of how to better do this?